src/types/index.d.ts

```diff
@@ -15,7 +15,13 @@
  * each element converted to a byte); SQLite always returns blob data as
  * `Uint8Array`
  */
-type SQLiteCompatibleType = number | string | Uint8Array | Array<number> | bigint | null;
+type SQLiteCompatibleType =
```

Comment:
Formatting here might make it difficult to keep in sync with the upstream repository. Speaking of which, it would be nice to pull in the latest changes from `upstream` before merging this. This can be in a separate PR.

Reply:
I've reverted the changes